inference-engine/src/legacy_api/src/convert_function_to_cnn_network.cpp

```diff
@@ -937,6 +939,8 @@ void convertFunctionToICNNNetwork(const std::shared_ptr<const ::ngraph::Function
 std::shared_ptr<CNNNetworkImpl> convertFunctionToICNNNetwork(const std::shared_ptr<const ::ngraph::Function> &graph,
                                                              const ICNNNetwork &network,
                                                              bool keep_constant_inputs) {
+    OV_ITT_SCOPED_TASK(itt::domains::IELegacy, "details::convertFunctionToICNNNetwork");
```

Comment:
do we need it here? this `convertFunctionToICNNNetwork` invokes the `convertFunctionToICNNNetwork` above, so the functions is traced using the above `convertFunctionToICNNNetwork`

Reply:
ok